howtos/how_to_request_cached_data.py

```diff
@@ -0,0 +1,139 @@
+"""
+This module contains example snippets on how to request cached data from a direct receiver. These examples assume
```

Comment:
Requesting cached data from a Cache via the direct receiver, not getting the data from the receiver

Reply:
Please review fix